chkRanges.c

```diff
@@ -0,0 +1,86 @@
+#include "chkRanges.h"
+
+int ranges = 0;
+int chrgSession[] = {3, 3, 5, 4, 10, 11, 12};
+int rangeCount[3] = {1, 1, 1};
+struct data
+{
+    int id;
+    char range[10];
+    int count;
+}s;
+
+void chkrange() {
```

Comment:
`chkrange` and `checkRange` mean the same thing... but the functions are different. Name the functions by their purpose.

Reply:
Yes, it's all corrected.